app/src/main/java/com/example/lister/MainActivity.java

```diff
@@ -276,6 +294,15 @@ private static double getPrice(String message) {
                     continue;
                 }
             }
+            if(s.length() > 0 && s.charAt(0) == '$') {
+                try {
+                    price = Double.parseDouble(s.substring(1));
+                    Log.d("Price Parse", "Price found from $: " + price);
+                    break;
+                } catch (Exception e) {
+                    continue;
+                }
+            }
```

Comment:
I tried adding more logic but it didn't help my case. I may look more at this tomorrow (Monday) so we can maybe actually use real price tags instead of hand written things for the demo. For me, with real price tags it sets the price as the number of ounces instead of the price.

Reply:
Maybe depends on the picture, works fine for me with this one: https://i.pinimg.com/originals/e2/7b/5f/e27b5f56f8d3525574a16a06557eeab7.jpg

Reply:
ooo sick thanks! I'll use that for the demo then assuming that works for me too. ill try it in a sec